lib/mfem/SampleMesh.cpp

```diff
@@ -316,8 +328,10 @@ void BuildSampleMesh(ParMesh& pmesh, vector<ParFiniteElementSpace*> & fespace,
 
     const int d = pmesh.Dimension();
 
-    H1_FECollection h1_coll(1, d);  // Must be first order, to get a bijection between vertices and DOF's.
-    ParFiniteElementSpace H1DummySpace(&pmesh, &h1_coll);  // This constructor effectively sets vertex (DOF) global indices.
+    H1_FECollection h1_coll(1,
```

Comment:
This was split by astyle, but it would be better to put the comment on the previous line, to shorten the lines.